internal/scorecard/plugins/olm_tests.go

```diff
@@ -176,6 +181,42 @@ func NewOLMTestSuite(conf OLMTestConfig) *schelpers.TestSuite {
 
 // Test Implentations
 
+// Run - implements Test interface
+func (t *BundleValidationTest) Run(ctx context.Context) *schelpers.TestResult {
+	res := &schelpers.TestResult{Test: t, MaximumPoints: 1}
+
+	if t.OLMTestConfig.Bundle == "" {
+		res.Errors = append(res.Errors, errors.New("add a OLM 'bundle' directory which is required for this test"))
+		return res
+	}
+
+	// Get the validation API log because it contains
+	// validation output that we include into the scorecard test output.
+	validationLogOutput := new(bytes.Buffer)
+	origOutput := logrus.StandardLogger().Out
+	logrus.SetOutput(validationLogOutput)
```

Comment:
why I new lib for logs was added? Is not it using another lib already?

Reply:
the validation API uses logrus, to get at its log, the logrus package is used.

Reply:
I am not sure if here we should not keep using the default libs to log.

Reply:
what those lines of code do is capture the logs produced by the API when it reads a bundle, those logs are picked up using logrus because the API logs using logrus, the log output is then added into the TestResult log so users can see the logs being produced by the bundle validation test.  In this PR, the Log field is added to test results so that in the future, any test that produces a log has a place to put log output into the scorecard test result.